module/sockets/socket-event.mjs

```diff
@@ -0,0 +1,33 @@
+/**
+ * @typedef {object} EventData
+ * @property {string} event     The unique event data name.
+ * @property {string[]} userIds     The ids of users that should perform the operation.
+ */
+
+export default class SocketEvent {
+  /**
+   * The socket event name, which must be unique and is used to call the event.
+   * @type {string}
+   */
+  static eventName;
```

Comment:
Might be cleaner to have these be non-static methods. Lemme know what you think.

Reply:
I think this works fine, the only real thing that would get us would be losing the `static` keywords a few places, but we'd have to create an instance during registration in that case, so it would be a trade-off.